rawpointer.go

```diff
@@ -56,7 +56,14 @@ func rawDoubleFarPointer(segID SegmentID, off Address) rawPointer {
 // a near pointer in the destination segment.  Its offset will be
 // relative to the beginning of the segment.
 func landingPadNearPointer(far, tag rawPointer) rawPointer {
-	return tag | rawPointer(far.farAddress()-Address(wordSize))<<2
+	farFarAddress := far.farAddress() / Address(wordSize)
```

Comment:
Since this variable is not a byte address (it's not a word address), let's inline this into the next line.